app/app.go

```diff
@@ -42,6 +42,7 @@ import (
 	"github.com/mendersoftware/deployments/store"
 	"github.com/mendersoftware/deployments/store/mongo"
 	"github.com/mendersoftware/deployments/utils"
+	mdriver "go.mongodb.org/mongo-driver/mongo"
```

Comment:
As an (internal) developer guideline, the application layer should not import the database driver of the database layer (`store`).
```suggestion
```